frontend/pages/ManageControlsPage/OSUpdates/components/TargetSection/TargetSection.tsx

```diff
@@ -6,37 +6,48 @@ import { IConfig } from "interfaces/config";
 import SectionHeader from "components/SectionHeader";
 import Spinner from "components/Spinner";
 
-import MacOSTargetForm from "../MacOSTargetForm";
 import WindowsTargetForm from "../WindowsTargetForm";
 import PlatformTabs from "../PlatformTabs";
 import { OSUpdatesSupportedPlatform } from "../../OSUpdates";
 
 const baseClass = "os-updates-target-section";
 
 type GetDefaultFnParams = {
+  osType?: "darwin" | "ios" | "ipados";
   currentTeamId: number;
   appConfig: IConfig;
   teamConfig?: ITeamConfig;
 };
 
-const getDefaultMacOSVersion = ({
+const getDefaultOSVersion = ({
+  osType,
   currentTeamId,
   appConfig,
   teamConfig,
 }: GetDefaultFnParams) => {
-  return currentTeamId === API_NO_TEAM_ID
-    ? appConfig?.mdm.macos_updates.minimum_version ?? ""
-    : teamConfig?.mdm?.macos_updates.minimum_version ?? "";
+  const mdmData =
+    currentTeamId === API_NO_TEAM_ID ? appConfig?.mdm : teamConfig?.mdm;
+
+  if (osType === "darwin") return mdmData?.macos_updates.minimum_version ?? "";
```

Comment:
Maybe we can use `APPLE_PLATFORMS_TO_CONFIG_FIELDS` again to clean this up, something like:
```
return osType && mdmData
  ? mdmData[APPLE_PLATFORMS_TO_CONFIG_FIELDS[osType]].minimum_version ?? ""
  : "";
```

Reply:
Yo yo, tried to implement this but the typeguards started getting more complicated than it's worth.

Ex: It would need to be written as

``` return osType && mdmData
  ? mdmData[APPLE_PLATFORMS_TO_CONFIG_FIELDS[osType] as keyof typeof mdmData].minimum_version ?? ""
  : ""; ```
  And even then its upset because not all fields have minimum_version.

Reply:
Does exporting the mapping `as const` solve the issue?

```
export const APPLE_PLATFORMS_TO_CONFIG_FIELDS = {
  darwin: "macos_updates",
  ios: "ios_updates",
  ipados: "ipados_updates",
} as const;
```

Seems to work for me like that:
```
const getDefaultOSVersion = ({
  osType,
  currentTeamId,
  appConfig,
  teamConfig,
}: GetDefaultFnParams) => {
  const mdmData =
    currentTeamId === API_NO_TEAM_ID ? appConfig?.mdm : teamConfig?.mdm;

  return osType && mdmData
    ? mdmData[APPLE_PLATFORMS_TO_CONFIG_FIELDS[osType]].minimum_version ?? ""
    : "";
};
```

also might be good to move the export to a more central location, if not too much trouble